code/Properties/launchSettings.json

```diff
@@ -1,9 +1,9 @@
 {
-    "profiles": {
-        "ttt-reborn": {
-            "commandName": "Executable",
-            "executablePath": "C:\\Program Files (x86)\\Steam\\steamapps\\common\\sbox\\sbox.exe",
-            "commandLineArgs": "+developer +sw -noassert -tools"
-        }
+  "profiles": {
+    "ttt-reborn": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Program Files (x86)\\Steam\\steamapps\\common\\sbox\\sbox.exe",
+      "commandLineArgs": "\u002Bdeveloper \u002Bsw -noassert -tools"
     }
-}
+  }
+}
```

Comment:
This isn't needed as well :D

Reply:
I don't think the `.gitignore` will ignore until the file is committed. So will need to delete this file after the merge.